packages/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -4436,14 +4436,17 @@ export default class Meeting extends StatelessWebexPlugin {
   };
 
   createMediaConnection(turnServerInfo) {
-    const mc = Media.createMediaConnection(this.mediaProperties, {
-      isMultistream: this.isMultistream,
-      meetingId: this.id,
-      remoteQualityLevel: this.mediaProperties.remoteQualityLevel,
-      enableRtx: this.config.enableRtx,
-      enableExtmap: this.config.enableExtmap,
-      turnServerInfo
-    });
+    const mc = Media.createMediaConnection(
+      this.isMultistream,
+      `MC-${this.id.substring(0, 4)}`,
```

Comment:
I wanted to keep it short (hence only 4 chars from the id) and also easy to find if you don't know the meeting id (hence the fixed "MC-" part)